src/XMakeTasks/LC.cs

```diff
@@ -178,15 +178,15 @@ private void AddCommands(CommandLineBuilderExtension commandLine)
 
         /// <summary>
         /// Generates response file with arguments for lc.exe
-        /// Used when targeting framework version is 4.0 or later
+        /// Used when targeting framework version is 4.6.2 or later
         /// </summary>
         /// <param name="commandLine">command line builder class to add arguments to the response file</param>
         protected internal override void AddResponseFileCommands(CommandLineBuilderExtension commandLine)
         {
             Version targetFramework = Util.GetTargetFrameworkVersion(TargetFrameworkVersion);
-            // Don't generate response file on versions of the framework < 4.0
+            // Don't generate response file on versions of the framework < 4.6.2
             // They will use the 2.x SDK lc.exe which does not understand response files
-            if (targetFramework.CompareTo(new Version("4.0")) < 0)
+            if (targetFramework.CompareTo(new Version("4.6.2")) < 0)
```

Comment:
I think the right version here is 4.6. I explained my reasoning in https://github.com/Microsoft/msbuild/issues/914#issuecomment-264290978.